examples/addmod.rs

```diff
@@ -0,0 +1,92 @@
+//! Addmod example for i64, i32, u64, u32.
+#![cfg_attr(target_arch = "wasm32", no_std)]
+#![cfg_attr(target_arch = "wasm32", no_main)]
+
+extern crate zink;
+use zink::primitives::{numeric::Numeric, U256};
+
+#[zink::external]
+pub fn addmod_i32(a: i32, b: i32, n: i32) -> i32 {
+    a.addmod(b, n)
+}
+
+#[zink::external]
+pub fn addmod_i64(a: i64, b: i64, n: i64) -> i64 {
+    a.addmod(b, n)
+}
+
+#[zink::external]
+pub fn addmod_u32(a: u32, b: u32, n: u32) -> u32 {
+    a.addmod(b, n)
+}
+
+#[zink::external]
+pub fn addmod_u64(a: u64, b: u64, n: u64) -> u64 {
+    a.addmod(b, n)
+}
+
+#[zink::external]
+pub fn addmod_U256(a: U256, b: U256, n: U256) -> U256 {
+    a.addmod(b, n)
+}
+
+#[cfg(not(target_arch = "wasm32"))]
+fn main() {}
+
+// FIXME: this test should pass, fix it to learn how to
+// add opcodes to the compiler
+#[ignore]
```

Comment:
```suggestion
```
why the test is still ignored?

Reply:
https://github.com/zink-lang/zink/pull/283#discussion_r1861750625